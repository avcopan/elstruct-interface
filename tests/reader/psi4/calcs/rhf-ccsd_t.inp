# Psi4 1.0 Computation 

molecule {
 0 1
 O          0.0000000000       -0.0530229169        0.0000000000
 H          0.0000000000        0.5556614584       -0.7504910643
 H          0.0000000000        0.5556614584        0.7504910643
 units angstrom
}

set reference rhf
set scf_type pk
set basis 6-31G*

energy('ccsd(t)')
