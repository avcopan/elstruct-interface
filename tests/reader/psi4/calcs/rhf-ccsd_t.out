
    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                               Psi4 1.2.1 release

                         Git: Rev {HEAD} 406f4de 


    R. M. Parrish, L. A. Burns, D. G. A. Smith, A. C. Simmonett,
    A. E. DePrince III, E. G. Hohenstein, U. Bozkaya, A. Yu. Sokolov,
    R. Di Remigio, R. M. Richard, J. F. Gonthier, A. M. James,
    H. R. McAlexander, A. Kumar, M. Saitow, X. Wang, B. P. Pritchard,
    P. Verma, H. F. Schaefer III, K. Patkowski, R. A. King, E. F. Valeev,
    F. A. Evangelista, J. M. Turney, T. D. Crawford, and C. D. Sherrill,
    J. Chem. Theory Comput. 13(7) pp 3185--3197 (2017).
    (doi: 10.1021/acs.jctc.7b00174)


                         Additional Contributions by
    P. Kraus, H. Kruse, M. H. Lechner, M. C. Schieber, and R. A. Shaw

    -----------------------------------------------------------------------


    Psi4 started on: Tuesday, 15 January 2019 11:43AM

    Process ID: 89364
    Host:       blogin3
    PSIDATADIR: /home/kmoore/miniconda/envs/esenv/share/psi4
    Memory:     500.0 MiB
    Threads:    1
    
  ==> Input File <==

--------------------------------------------------------------------------
# Psi4 1.0 Computation 

molecule {
 0 1
 O          0.0000000000       -0.0530229169        0.0000000000
 H          0.0000000000        0.5556614584       -0.7504910643
 H          0.0000000000        0.5556614584        0.7504910643
 units angstrom
}

set reference rhf
set scf_type pk
set basis 6-31G*

energy('ccsd(t)')
--------------------------------------------------------------------------

*** tstart() called on blogin3
*** at Tue Jan 15 11:43:40 2019

   => Loading Basis Set <=

    Name: 6-31G*
    Role: ORBITAL
    Keyword: BASIS
    atoms 1   entry O          line   145 file /home/kmoore/miniconda/envs/esenv/share/psi4/basis/6-31gs.gbs 
    atoms 2-3 entry H          line    44 file /home/kmoore/miniconda/envs/esenv/share/psi4/basis/6-31gs.gbs 


         ---------------------------------------------------------
                                   SCF
            by Justin Turney, Rob Parrish, Andy Simmonett
                             and Daniel Smith
                              RHF Reference
                        1 Threads,    500 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c2v
    Full point group: C2v

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
         O            0.000000000000     0.000000000000    -0.068120834725    15.994914619560
         H           -0.000000000000     0.750491064300     0.540563540575     1.007825032070
         H            0.000000000000    -0.750491064300     0.540563540575     1.007825032070

  Running in c2v symmetry.

  Rotational constants: A =     25.41806  B =     14.84876  C =      9.37315 [cm^-1]
  Rotational constants: A = 762014.35919  B = 445154.74598  C = 280999.82600 [MHz]
  Nuclear repulsion =    9.114682954848963

  Charge       = 0
  Multiplicity = 1
  Electrons    = 10
  Nalpha       = 5
  Nbeta        = 5

  ==> Algorithm <==

  SCF Algorithm Type is PK.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is SAD.
  Energy threshold   = 1.00e-08
  Density threshold  = 1.00e-08
  Integral threshold = 0.00e+00

  ==> Primary Basis <==

  Basis Set: 6-31G*
    Blend: 6-31G*
    Number of shells: 10
    Number of basis function: 19
    Number of Cartesian functions: 19
    Spherical Harmonics?: false
    Max angular momentum: 2

  ==> Pre-Iterations <==

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     A1        10      10       0       0       0       0
     A2         1       1       0       0       0       0
     B1         3       3       0       0       0       0
     B2         5       5       0       0       0       0
   -------------------------------------------------------
    Total      19      19       5       5       5       0
   -------------------------------------------------------

  ==> Integral Setup <==

  Using in-core PK algorithm.
   Calculation information:
      Number of atoms:                   3
      Number of AO shells:              10
      Number of primitives:             23
      Number of atomic orbitals:        19
      Number of basis functions:        19

      Integral cutoff                 1.00e-12
      Number of threads:                 1

  Performing in-core PK
  Using 36290 doubles for integral storage.
  We computed 1540 shell quartets total.
  Whereas there are 1540 unique shell quartets.

  ==> DiskJK: Disk-Based J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    Memory (MB):               375
    Schwarz Cutoff:          1E-12

    OpenMP threads:              1

  Minimum eigenvalue in the overlap matrix is 2.2607421967E-02.
  Using Symmetric Orthogonalization.

  SCF Guess: Superposition of Atomic Densities via on-the-fly atomic UHF.

  ==> Iterations <==

                        Total Energy        Delta E     RMS |[F,P]|

   @RHF iter   0:   -75.91261869086694   -7.59126e+01   1.42930e-01 
   @RHF iter   1:   -75.97245904420981   -5.98404e-02   2.49120e-02 
   @RHF iter   2:   -76.00147946165313   -2.90204e-02   1.29762e-02 DIIS
   @RHF iter   3:   -76.00931816810842   -7.83871e-03   1.92384e-03 DIIS
   @RHF iter   4:   -76.00969933005094   -3.81162e-04   5.25909e-04 DIIS
   @RHF iter   5:   -76.00974461091934   -4.52809e-05   1.13726e-04 DIIS
   @RHF iter   6:   -76.00974720849750   -2.59758e-06   1.64023e-05 DIIS
   @RHF iter   7:   -76.00974725401173   -4.55142e-08   2.80244e-06 DIIS
   @RHF iter   8:   -76.00974725524675   -1.23502e-09   3.92977e-07 DIIS
   @RHF iter   9:   -76.00974725526831   -2.15579e-11   3.68464e-08 DIIS
   @RHF iter  10:   -76.00974725526852   -2.13163e-13   4.39402e-09 DIIS

  ==> Post-Iterations <==

    Orbital Energies [Eh]
    ---------------------

    Doubly Occupied:                                                      

       1A1   -20.563638     2A1    -1.339399     1B2    -0.697137  
       3A1    -0.574732     1B1    -0.498367  

    Virtual:                                                              

       4A1     0.208352     2B2     0.301370     3B2     1.007331  
       5A1     1.131407     2B1     1.167057     6A1     1.170702  
       4B2     1.375632     7A1     1.443863     8A1     2.015007  
       1A2     2.037963     3B1     2.066297     9A1     2.610630  
       5B2     2.926503    10A1     3.960833  

    Final Occupation by Irrep:
             A1    A2    B1    B2 
    DOCC [     3,    0,    1,    1 ]

  Energy converged.

  @RHF Final Energy:   -76.00974725526852

   => Energetics <=

    Nuclear Repulsion Energy =              9.1146829548489627
    One-Electron Energy =                -122.9023153175880623
    Two-Electron Energy =                  37.7778851074705813
    Total Energy =                        -76.0097472552685218



Properties will be evaluated at   0.000000,   0.000000,   0.000000 [a0]

Properties computed using the SCF density matrix

  Nuclear Dipole Moment: [e a0]
     X:     0.0000      Y:     0.0000      Z:     1.0132

  Electronic Dipole Moment: [e a0]
     X:     0.0000      Y:     0.0000      Z:    -0.1171

  Dipole Moment: [e a0]
     X:     0.0000      Y:     0.0000      Z:     0.8961     Total:     0.8961

  Dipole Moment: [D]
     X:     0.0000      Y:     0.0000      Z:     2.2776     Total:     2.2776


*** tstop() called on blogin3 at Tue Jan 15 11:43:41 2019
Module time:
	user time   =       0.40 seconds =       0.01 minutes
	system time =       0.01 seconds =       0.00 minutes
	total time  =          1 seconds =       0.02 minutes
Total time:
	user time   =       0.40 seconds =       0.01 minutes
	system time =       0.01 seconds =       0.00 minutes
	total time  =          1 seconds =       0.02 minutes
 MINTS: Wrapper to libmints.
   by Justin Turney

   Calculation information:
      Number of threads:                 1
      Number of atoms:                   3
      Number of AO shells:              10
      Number of SO shells:               8
      Number of primitives:             23
      Number of atomic orbitals:        19
      Number of basis functions:        19

      Number of irreps:                  4
      Integral cutoff                 0.00e+00
      Number of functions per irrep: [  10    1    3    5 ]

 OEINTS: Overlap, kinetic, potential, dipole, and quadrupole integrals
         stored in file 35.

      Computing two-electron integrals...done
      Computed 4443 non-zero two-electron integrals.
        Stored in file 33.


*** tstart() called on blogin3
*** at Tue Jan 15 11:43:41 2019


	Wfn Parameters:
	--------------------
	Wavefunction         = CCSD_T
	Number of irreps     = 4
	Number of MOs        = 19
	Number of active MOs = 19
	AO-Basis             = NONE
	Semicanonical        = false
	Reference            = RHF
	Print Level          = 1

	IRREP	# MOs	# FZDC	# DOCC	# SOCC	# VIRT	# FZVR
	-----	-----	------	------	------	------	------
	 A1	   10	    0	    3	    0	    7	    0
	 A2	   1	    0	    0	    0	    1	    0
	 B1	   3	    0	    1	    0	    2	    0
	 B2	   5	    0	    1	    0	    4	    0
	Transforming integrals...
	IWL integrals will be deleted.
	(OO|OO)...
	Presorting SO-basis two-electron integrals.
	Sorting File: SO Ints (nn|nn) nbuckets = 1
	Transforming the one-electron integrals and constructing Fock matrices
	Starting first half-transformation.
	Sorting half-transformed integrals.
	First half integral transformation complete.
	Starting second half-transformation.
	Two-electron integral transformation complete.
	(OO|OV)...
	Starting second half-transformation.
	Two-electron integral transformation complete.
	(OO|VV)...
	Starting second half-transformation.
	Two-electron integral transformation complete.
	(OV|OO)...
	Starting first half-transformation.
	Sorting half-transformed integrals.
	First half integral transformation complete.
	Starting second half-transformation.
	Two-electron integral transformation complete.
	(OV|OV)...
	Starting second half-transformation.
	Two-electron integral transformation complete.
	(OV|VV)...
	Starting second half-transformation.
	Two-electron integral transformation complete.
	(VV|OO)...
	Starting first half-transformation.
	Sorting half-transformed integrals.
	First half integral transformation complete.
	Starting second half-transformation.
	Two-electron integral transformation complete.
	(VV|OV)...
	Starting second half-transformation.
	Two-electron integral transformation complete.
	(VV|VV)...
	Starting second half-transformation.
	Two-electron integral transformation complete.
	Frozen core energy     =      0.00000000000000

	Size of irrep 0 of <ab|cd> integrals:      0.005 (MW) /      0.039 (MB)
	Size of irrep 1 of <ab|cd> integrals:      0.001 (MW) /      0.007 (MB)
	Size of irrep 2 of <ab|cd> integrals:      0.001 (MW) /      0.010 (MB)
	Size of irrep 3 of <ab|cd> integrals:      0.004 (MW) /      0.029 (MB)
	Total:                                     0.011 (MW) /      0.086 (MB)

	Size of irrep 0 of <ia|bc> integrals:      0.002 (MW) /      0.015 (MB)
	Size of irrep 1 of <ia|bc> integrals:      0.000 (MW) /      0.002 (MB)
	Size of irrep 2 of <ia|bc> integrals:      0.001 (MW) /      0.004 (MB)
	Size of irrep 3 of <ia|bc> integrals:      0.001 (MW) /      0.010 (MB)
	Total:                                     0.004 (MW) /      0.031 (MB)

	Size of irrep 0 of tijab amplitudes:       0.001 (MW) /      0.006 (MB)
	Size of irrep 1 of tijab amplitudes:       0.000 (MW) /      0.000 (MB)
	Size of irrep 2 of tijab amplitudes:       0.000 (MW) /      0.002 (MB)
	Size of irrep 3 of tijab amplitudes:       0.000 (MW) /      0.003 (MB)
	Total:                                     0.001 (MW) /      0.011 (MB)

	Nuclear Rep. energy          =      9.11468295484896
	SCF energy                   =    -76.00974725526852
	One-electron energy          =   -122.90231528093493
	Two-electron energy          =     37.77788507081747
	Reference energy             =    -76.00974725526849

*** tstop() called on blogin3 at Tue Jan 15 11:43:41 2019
Module time:
	user time   =       0.02 seconds =       0.00 minutes
	system time =       0.02 seconds =       0.00 minutes
	total time  =          0 seconds =       0.00 minutes
Total time:
	user time   =       0.45 seconds =       0.01 minutes
	system time =       0.03 seconds =       0.00 minutes
	total time  =          1 seconds =       0.02 minutes

*** tstart() called on blogin3
*** at Tue Jan 15 11:43:41 2019

            **************************
            *                        *
            *        CCENERGY        *
            *                        *
            **************************

    Nuclear Rep. energy (wfn)     =    9.114682954848963
    SCF energy          (wfn)     =  -76.009747255268522
    Reference energy    (file100) =  -76.009747255268493

    Input parameters:
    -----------------
    Wave function   =     CCSD_T
    Reference wfn   =     RHF
    Brueckner       =     No
    Memory (Mbytes) =     524.3
    Maxiter         =     50
    R_Convergence   =     1.0e-07
    E_Convergence   =     1.0e-06
    Restart         =     Yes
    DIIS            =     Yes
    AO Basis        =     NONE
    ABCD            =     NEW
    Cache Level     =     2
    Cache Type      =     LOW
    Print Level     =     1
    Num. of threads =     1
    # Amps to Print =     10
    Print MP2 Amps? =     No
    Analyze T2 Amps =     No
    Print Pair Ener =     No
    Local CC        =     No
    SCS-MP2         =     False
    SCSN-MP2        =     False
    SCS-CCSD        =     False

MP2 correlation energy -0.1893545440602351
                Solving CC Amplitude Equations
                ------------------------------
  Iter             Energy              RMS        T1Diag      D1Diag    New D1Diag    D2Diag
  ----     ---------------------    ---------   ----------  ----------  ----------   --------
     0        -0.189354544060235    0.000e+00    0.000000    0.000000    0.000000    0.107276
     1        -0.193702563458965    2.747e-02    0.004104    0.007651    0.007651    0.115018
     2        -0.197692876352177    9.889e-03    0.004847    0.008835    0.008835    0.120931
     3        -0.198132316720228    2.501e-03    0.005250    0.010022    0.010022    0.122296
     4        -0.198153730444634    6.017e-04    0.005283    0.010244    0.010244    0.122519
     5        -0.198167589444965    1.761e-04    0.005309    0.010380    0.010380    0.122558
     6        -0.198164486936652    5.547e-05    0.005319    0.010433    0.010433    0.122548
     7        -0.198164055564865    1.416e-05    0.005322    0.010448    0.010448    0.122546
     8        -0.198164009927526    2.689e-06    0.005322    0.010451    0.010451    0.122545
     9        -0.198163952167832    5.627e-07    0.005322    0.010451    0.010451    0.122545
    10        -0.198163965903516    1.078e-07    0.005322    0.010451    0.010451    0.122545
    11        -0.198163964264131    2.234e-08    0.005322    0.010451    0.010451    0.122545

    Iterations converged.


    Largest TIA Amplitudes:
              2   0         0.0089627243
              4  12         0.0073201061
              3   8        -0.0069587924
              4  10        -0.0046150952
              4  13         0.0037950559
              1   1         0.0036884201
              2   5         0.0032071094
              1   2         0.0031741932
              1   6         0.0029789352
              1   0        -0.0024923326

    Largest TIjAb Amplitudes:
      3   3   8   8        -0.0522225958
      4   4  10  10        -0.0406018296
      2   2   2   2        -0.0293581665
      4   4  11  11        -0.0281984501
      4   4  12  12        -0.0280189619
      3   4   8  12        -0.0263683277
      4   3  12   8        -0.0263683277
      4   4  10  12        -0.0260662317
      4   4  12  10        -0.0260662317
      3   4   8  10        -0.0245159301

    SCF energy       (wfn)                    =  -76.009747255268522
    Reference energy (file100)                =  -76.009747255268493

    Opposite-spin MP2 correlation energy      =   -0.140886879769367
    Same-spin MP2 correlation energy          =   -0.048467664290868
    MP2 correlation energy                    =   -0.189354544060235
      * MP2 total energy                      =  -76.199101799328730

    Opposite-spin CCSD correlation energy     =   -0.155487910048054
    Same-spin CCSD correlation energy         =   -0.042676054451039
    CCSD correlation energy                   =   -0.198163964264131
      * CCSD total energy                     =  -76.207911219532619


*** tstop() called on blogin3 at Tue Jan 15 11:43:41 2019
Module time:
	user time   =       0.14 seconds =       0.00 minutes
	system time =       0.18 seconds =       0.00 minutes
	total time  =          0 seconds =       0.00 minutes
Total time:
	user time   =       0.59 seconds =       0.01 minutes
	system time =       0.21 seconds =       0.00 minutes
	total time  =          1 seconds =       0.02 minutes

*** tstart() called on blogin3
*** at Tue Jan 15 11:43:41 2019

            **************************
            *                        *
            *        CCTRIPLES       *
            *                        *
            **************************


    Wave function   =    CCSD_T
    Reference wfn   =      RHF

    Nuclear Rep. energy (wfn)                =    9.114682954848963
    SCF energy          (wfn)                =  -76.009747255268522
    Reference energy    (file100)            =  -76.009747255268493
    CCSD energy         (file100)            =   -0.198163964264131
    Total CCSD energy   (file100)            =  -76.207911219532619

    Number of ijk index combinations:               35
    Memory available in words        :        65536000
    ~Words needed per explicit thread:            1372
    Number of threads for explicit ijk threading:    1

    MKL num_threads set to 1 for explicit threading.

    (T) energy                                =   -0.001941474315526
      * CCSD(T) total energy                  =  -76.209852693848148


*** tstop() called on blogin3 at Tue Jan 15 11:43:41 2019
Module time:
	user time   =       0.02 seconds =       0.00 minutes
	system time =       0.01 seconds =       0.00 minutes
	total time  =          0 seconds =       0.00 minutes
Total time:
	user time   =       0.61 seconds =       0.01 minutes
	system time =       0.22 seconds =       0.00 minutes
	total time  =          1 seconds =       0.02 minutes

    Psi4 stopped on: Tuesday, 15 January 2019 11:43AM
    Psi4 wall time for execution: 0:00:01.12

*** Psi4 exiting successfully. Buy a developer a beer!
